scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -0,0 +1,540 @@
+
+{% macro gtcall_panel(variant) %}
+  <div class="card panel-default">
+    <div class="panel-heading">
+      GT call
+      {% if variant.is_par %}
+        <span class="badge badge-info">PAR</span>
+      {% endif %}
+    </div>
+    <div class="card-body">
+    <div class="table-responsive">
+      <table class="table table-bordered">
+        <thead>
+          <tr>
+            <th rowspan="2">Sample</th>
+            <th rowspan="2">Genotype (GT)</th>
+            <th colspan="2" title="Unfiltered count of reads that support a given allele.">Allele depth (AD)</th>
+            <th rowspan="2" colspan="1" title="Phred-scaled confidence that the true genotype is the one provided in GT (max=99).">Genotype quality (GQ)</th>
+            <tr>
+              <th>Reference</th>
+              <th>Alternative</th>
+            </tr>
+          </tr>
+        </thead>
+        <tbody>
+          {% for sample in variant.samples %}
+            <tr {{ 'class="danger"' if sample.is_affected }}>
+              <td>{{ sample.display_name }}</td>
+              <td class="text-center">{{ sample.genotype_call }}</td>
+              {% if sample.allele_depths %}
+                  {% for number in sample.allele_depths %}
+                    <td class="text-right">{{ number }}</td>
+                  {% endfor %}
+              {% else %}
+                  <td class="text-right"><small>N/A</small></td>
+                  <td class="text-right"><small>N/A</small></td>
+              {% endif %}
+              <td class="text-right">{{ sample.genotype_quality }}</td>
+            </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+    </div>
+    <div class="panel-footer">
+      {% for name, caller in variant.callers %}
+        <span class="badge badge-secondary">{{ name }}: {{ caller }}</span>
+      {% endfor %}
+    </div>
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro rankscore_panel(variant) %}
+  <div class="card panel-default">
+    <div class="panel-heading">Rank score results</div>
+    <div class="table-responsive">
+      <table class="table table-bordered">
+        <thead>
+          <tr>
+            {% for result in variant.rank_score_results %}
+              <th><small>{{ result.category|replace('_', ' ') }}</small></th>
+            {% endfor %}
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            {% for result in variant.rank_score_results %}
+              <td class="text-right">
+                {{ result.score }}
+              </td>
+            {% endfor %}
+          <tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro overlapping_panel(variant, overlapping_vars, case, institute) %}
+  <div class="card panel-default">
+    {% if variant.category == 'sv' %}
+      <div class="panel-heading">Overlapping SNVs</div>
+    {% else %}
+      <div class="panel-heading">Overlapping SVs</div>
+    {% endif %}
+    <div class="table-responsive">
```

Comment:
body? The margins are I suppose a matter of taste; perhaps we could have margin-less class for the card body as well, if we want the cards to be complete, but no margins?